crates/core/c8y_api/Cargo.toml

```diff
@@ -35,6 +35,7 @@ tracing = { version = "0.1", features = ["attributes", "log"] }
 anyhow = "1.0"
 assert-json-diff = "2.0"
 assert_matches = "1.5"
+maplit = "1.0.2"
```

Comment:
This is fine as a dev dependency. But just FYI: you could statically create HashMap using the `from` method as follows:

```
let solar_distance = HashMap::from([
    ("Mercury", 0.4),
    ("Venus", 0.7),
    ("Earth", 1.0),
    ("Mars", 1.5),
]);
```